src/OpenTelemetry.SemanticConventions/OpenTelemetry.SemanticConventions.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <Description>OpenTelemetry Semantic Conventions</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
```

Comment:
lets remove "core-" now, else this will be versioned same as core packages, but as per spec, semantic conventions are not still marked experimental, so it should be released as pre-release.

Reply:
Ah I see. So, I guess just remove the MinVer tag altogether here and when we release this we use the pre-release tag?